### ISSUE TITLE
Devika doesn't Load in Safari on macOS Sonoma (works fine in chrome)

### ISSUE THREAD

**Issue by danielraffel** [stitionai/devika#426]
**Describe the bug**
Start Devika and open [localhost](localhost:3000/) in Safari and note the page does not load on macOS Sonoma 14.4.1 in Safari Version 17.4.1

**To Reproduce**
1. Start Devika
2. visit localhost:3000 in safari
3. note that the webpage is empty (tho there is source code if you view source)

**Expected behavior**
Devika will load a webpage at localhost:3000 in Safari

**Screenshots**
Empty browser window
<img width="1587" alt="Screenshot 2024-04-11 at 9 23 59 PM" src="https://github.com/stitionai/devika/assets/25807/555eecac-b5fc-440a-a0dd-d0eed5336768">

**Desktop (please complete the following information):**
 - OS: macOS Sonoma 14.4.1
 - Browser: Safari
 - Version: 17.4.1

**Additional context**
Devika running at localhost:3000 opens fine in chrome version 123.0.6312.107 on macOS

**Comment by NinjaEde** [comment]
I can confirm this behavior in Safari. Additionally I found an error in the browser console which could be the cause for the problem. Unfortunately the console doesn't says which line / file is causing this. 
![CleanShot 2024-04-12 at 15 50 19@2x](https://github.com/stitionai/devika/assets/8362096/0be78c47-c71d-481e-82f2-6eb59bf3c0e6)

I found in the UI log the following error: 
![CleanShot 2024-04-12 at 15 58 11@2x](https://github.com/stitionai/devika/assets/8362096/d0ae16e4-e3a8-4002-89e2-e6309826a492)
Could this eventually cause the problem?
